guides/v2.1/ext-best-practices/tutorials/dynamic-row-system-config.md

```diff
@@ -0,0 +1,79 @@
+---
+group: extension-best-practices
+subgroup: 02_Extension-Coding
+title: Creating a dynamic row system config
+menu_title: Creating a dynamic row system config
+menu_order: 1010
+functional_areas:
+  - Standards
+---
+
+### Overview
+
+This tutorial gives instructions for adding a new dynamic rows system config in the {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento admin{% endglossarytooltip %}, by extending the [Magento/Config/Block/System/Config/Form/Field/FieldArray/AbstractFieldArray][0]{:target="_blank"} class. 
+
+## Step 1: Add your new system field 
+{:#step-1}
+
+**etc/adminhtml/system.xml**
+
+```xml
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="general" translate="label" type="text">
+            <group id="quantity_ranges" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label>Quantity Ranges</label>
+                <field id="ranges" translate="label" sortOrder="5" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Ranges</label>
+                    <frontend_model>[vendor]\[module]\Block\Adminhtml\Form\Field\Ranges</frontend_model>
+                    <backend_model>Magento\Config\Model\Config\Backend\Serialized\ArraySerialized</backend_model>
+                </field>
+            </group>
+        </section>
+    </system>
+</config>
+```
+
+This code adds a new system config on the following location:
```

Comment:
```suggestion
This code adds a new system config in the following location:
```

Reply:
Also remove the spaces between the words and / in the path below.

Reply:
The path isn't showing correctly. When I click View File, I see this:

<img width="679" alt="Screen Shot 2019-05-21 at 8 05 05 AM" src="https://user-images.githubusercontent.com/14031342/58098426-31fa1400-7b9f-11e9-8ea2-6f1833831a12.png">